code/modules/mob/dead/observer/observer.dm

```diff
@@ -538,6 +538,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	if(!target)
 		return 0
+
+	if(istype (target, /mob/living/simple_animal/hostile/megafauna))
```

Comment:
omg why is this attitude resurfacing? oranges has said it, now you've said it.

A GENERIC solution is var nicer than a snowflake ```istype()```
YES it's a little cluttered to have a var for every little thing but it's also infinitely more maintainable.